frontend/src/container/Login.jsx

```diff
@@ -110,14 +114,19 @@ class Login extends React.Component {
 
     event.preventDefault()
 
-    const { email, password } = event.target
+    const { login, password } = event.target
 
-    if (email.value === '' || password.value === '') {
+    if (login.value === '' || password.value === '') {
       props.dispatch(newFlashMessage(props.t('Please enter a login and a password'), 'warning'))
       return
     }
 
-    const fetchPostUserLogin = await props.dispatch(postUserLogin(email.value, password.value, state.inputRememberMe))
+    const credentials = {
+      ...(checkEmailValidity(login.value) ? { email: login.value } : { username: login.value }),
```

Comment:
Are we sure we want to do this like that?
Are people allowed to use "@" in their username? This behavior would probably be annoying for them. A discussion needs to happen with the backend crowd on this I think.
Another solution would be to have two inputs but it would probably be a bit annoying. Or a checkbox that gets checked automatically, and the user can revert the check.

Reply:
Allowed chars in username are `azAZ09-_`, 
Yes other front solution are possible but I think it would be way better to keep just one input for both (email and username)